src/ert/config/workflow_job.py

```diff
@@ -143,11 +150,36 @@ class ErtScriptWorkflow(_WorkflowJob):
     Single workflow configuration object
     """
 
-    ert_script: type[ErtScript] = None  # type: ignore
+    ert_script: builtins.type[ErtScript] = None  # type: ignore
```

Comment:
Making `ert_script` serializable seems to not be related to the change here, and could probably easily be in its own commit.